syntax = "proto3";

package db_proto;

message Point {
	double x = 1;
	double y = 2;
}

message Color {
	string color_str = 1;
}
